scripts/ci/run-jenkins.sh

```diff
@@ -94,6 +94,23 @@ if [[ ${CI_TAGS} == 'product-sdks' ]];
 	   exit 0
 fi
 
+if [[ ${CI_TAGS} == 'webassembly' ]];
+   then
+	   echo "DISABLE_ANDROID=1" > sdks/Make.config
+	   echo "DISABLE_IOS=1" > sdks/Make.config
```

Comment:
This will not have the consequence you intend: `>` overrides the content of the file, so there won't be any `DISABLE_ANDROID=1` in this file, only the `DISABLE_IOS=1`

Reply:
🤦‍♂️